src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -287,6 +290,102 @@ def generate_support_proposals(self, comms_obj: MessagesData) -> Dict[str, str]:
                 final_messages[recipient] = str(suggested_proposals)
                 comms_obj.add_message(recipient, str(suggested_proposals))
         return final_messages
+        
+    def is_order_aggressive_to_powers(self, order: str, powers: List[str]):
+        """
+        check if the order is aggressive by
+        1. to attack allies unit
+        2. to move to allies' SC
+        3. support attack allies unit
+        4. support move to allies' SC
+
+        :param order: an order as string, e.g. "A BUD S F TRI"
+        :param powers: powers that we want to check if a bot is having aggressive move to
+        :return: Boolean
+
+        """
+        order_token = get_order_tokens(order)
+        if order_token[0][0] == "A" or order_token[0][0] == "F":
+            # for 1 and 2
```

Comment:
I would prefer writing out explicitly what these are rather than referencing docstring, so reader doesnt have to look back and forth (same for below).

Reply:
ok sure!